src/init.cpp

```diff
@@ -1488,6 +1488,13 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         uiInterface.NotifyBlockTip.disconnect(BlockNotifyGenesisWait);
     }
 
+#ifdef ENABLE_WALLET
```

Comment:
Yes I agree, let's merge this as it is a necessary bugfix, this can be improved later.